lemur/static/app/angular/sources/source/source.js

```diff
@@ -41,22 +41,14 @@ angular.module('lemur')
       PluginService.getByType('source').then(function (plugins) {
         $scope.plugins = plugins;
         _.each($scope.plugins, function (plugin) {
-          if (plugin.slug === $scope.source.pluginName) {
+          if (plugin.slug === $scope.source.plugin.slug) {
+            plugin.pluginOptions = $scope.source.plugin.pluginOptions;
```

Comment:
These fields were using incorrect variable names, resulting in the if statement never being executed and the plugin selection list showing up as empty.